### PR TITLE
RET-5160 -  Create Task "MultiplesReviewReferralResponseLegalOps"

### PR DESCRIPTION
**Pull request by marcjudge**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-5160

### Change description ###
Create Task "MultiplesReviewReferralResponseLegalOps"

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/wa-task-configuration-employment-et_englandwales_multiple.dmn [hunk 1]
```diff
@@ -177,6 +176,23 @@
           <text>true</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_1pom0v3">
+        <inputEntry id="UnaryTests_1hnt40n">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0kn2n9v">
+          <text>"MultiplesReviewReferralResponseLegalOps"</text>
```
Comment (fergusoshea) [inline src/main/resources/wa-task-configuration-employment-et_englandwales_multiple.dmn:184]: Can this be grouped with ReviewReferralLegalOpsMultiple below?

File: src/main/resources/wa-task-configuration-employment-et_scotland_multiple.dmn [hunk 2]
```diff
@@ -166,13 +183,13 @@
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_1faq2jo">
-          <text>"ReviewReferralAdminMultiple"</text>
+          <text>"MultiplesReviewReferralResponseLegalOps"</text>
```
Comment (fergusoshea) [inline src/main/resources/wa-task-configuration-employment-et_scotland_multiple.dmn:186]: Same as above.

File: src/main/resources/wa-task-initiation-employment-et_englandwales_multiple.dmn [hunk 3]
```diff
@@ -159,6 +159,35 @@ else null</text>
...
+          <text>"MultiplesReviewReferralResponseLegalOps"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0rfzea8">
+          <text>LATEST_REFERRAL_REPLY_TASK_NAME</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1cnlj1k">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0e2ak2s">
+          <text>"Processing"</text>
```
Comment (fergusoshea) [inline src/main/resources/wa-task-initiation-employment-et_englandwales_multiple.dmn:188]: Do we have an entry on Cancellation DMN for the Process Category Identifiers "Processing"?

File: src/main/resources/wa-task-initiation-employment-et_scotland_multiple.dmn [hunk 4]
```diff
@@ -159,6 +159,35 @@ else null</text>
...
+          <text>"MultiplesReviewReferralResponseLegalOps"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1adludt">
+          <text>LATEST_REFERRAL_REPLY_TASK_NAME</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_05qmv0d">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0z6yewn">
+          <text>"Processing"</text>
```
Comment (fergusoshea) [inline src/main/resources/wa-task-initiation-employment-et_scotland_multiple.dmn:188]: As above

**Review by fergusoshea — APPROVED** [review]

**Review by lvo700 — APPROVED** [review]

**Pull request merged**